diff --git i/src/ptCp.sh w/src/ptCp.sh
index 0be7e40..bc2b73d 100755
--- i/src/ptCp.sh
+++ w/src/ptCp.sh
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-source ~/.local/share/ptSh/config
-test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
+source $ZPFX/share/ptSh/config
 
 RESULT=$(cp "$@" 2>&1)
 
@@ -20,5 +19,5 @@ while read -r line; do
     else
         echo -ne "${SUCCESS_PREFIX_ESCAPE_CODES}${SUCCESS_PREFIX}\x1B[0m"
         echo -e "${SUCCESS_MESSAGE_ESCAPE_CODES}${SUCCESS_MESSAGE}\x1B[0m"
-    fi 
+    fi
 done <<< "$RESULT"
diff --git i/src/ptLs.sh w/src/ptLs.sh
index 73b90db..4941009 100755
--- i/src/ptLs.sh
+++ w/src/ptLs.sh
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-source ~/.local/share/ptSh/config
-test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
+source $ZPFX/share/ptSh/config
 
 columnSize=0
 IFS=' '
@@ -86,16 +85,16 @@ while read -r line; do
             nameLength=$((nameLength+1+${#words[$((5+nameWords))]}))
         done
     fi
-    
+
     nameLength=$((nameLength-1))
     filename="${filename::-1}"
     filename="$filename\x1B[0m"
-    
+
     actualChar=$((nameLength+prefixLength))
-    
+
     if [[ $1 == *"l"* ]]; then
         echo -ne $filename
-        align        
+        align
         echo -n "${words[0]} ${words[2]} ${words[3]}"
 
         if $link; then
diff --git i/src/ptMkdir.sh w/src/ptMkdir.sh
index 2c9d6a3..47391c7 100755
--- i/src/ptMkdir.sh
+++ w/src/ptMkdir.sh
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-source ~/.local/share/ptSh/config
-test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
+source $ZPFX/share/ptSh/config
 
 if [[ -z $1 ]] || [[ $1 == "--help" ]]; then
     cat <<< "$(mkdir --help)"
@@ -25,7 +24,7 @@ for name in "${names[@]}"; do
     echo -ne "${DIR_PREFIX_ESCAPE_CODES}${DIR_PREFIX}\x1B[0m"
     echo -e "${DIR_NAME_ESCAPE_CODES}$name\x1B[0m"
     ERR=$(mkdir $args $name 2>&1)
-    
+
     echo -n "└"
 
     while read -r line; do
diff --git i/src/ptPwd.sh w/src/ptPwd.sh
index 492ebc2..1b67597 100755
--- i/src/ptPwd.sh
+++ w/src/ptPwd.sh
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-source ~/.local/share/ptSh/config
-test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
+source $ZPFX/share/ptSh/config
 
 function setval { printf -v "$1" "%s" "$(cat)"; declare -p "$1"; }
 
diff --git i/src/ptTouch.sh w/src/ptTouch.sh
index b55d455..5391ab1 100755
--- i/src/ptTouch.sh
+++ w/src/ptTouch.sh
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-source ~/.local/share/ptSh/config
-test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
+source $ZPFX/share/ptSh/config
 
 if [[ -z $1 ]] || [[ $1 == "--help" ]]; then
     cat <<< "$(touch --help)"
