diff --git i/zsh-edit.plugin.zsh w/zsh-edit.plugin.zsh
index 2d06462..3216d3e 100644
--- i/zsh-edit.plugin.zsh
+++ w/zsh-edit.plugin.zsh
@@ -7,12 +7,10 @@ zmodload zsh/complist
   typeset -gHa _edit_opts=( extendedglob NO_listbeep NO_shortloops warncreateglobal )
   setopt $_edit_opts
 
-  local fdir=${${(%):-%x}:h}/functions
+  local fdir=${${(%):-%x}:A:h}/functions
   unfunction bindkey 2>/dev/null
-  autoload -Uz $fdir/bindkey $fdir/_*~*.zwc
-
-  bindkey -M emacs '^U'  backward-kill-line
-  bindkey -M emacs '^[e' redo
+  autoload -Uz $fdir/bindkey
+  autoload -Uz $fdir/_*~*.zwc(:t)
 
   .beginning-of-buffer() {
     CURSOR=0
