diff --git a/functions/_dirstack_ b/functions/_dirstack_
index fd459b5..b5e54af 100644
--- a/functions/_dirstack_
+++ b/functions/_dirstack_
@@ -1,7 +1,6 @@
 #autoload
 if [[ ! -v _comp_setup ]]; then
-  autoload -Uz compinit
-  compinit
+  zicompinit
 fi
 
 _dirstack_() {
@@ -11,4 +10,8 @@ _dirstack_() {
   compstate[list]='list force'
 }
 
+# _dirstack_() {
+#   dirs -lp | fzf
+# }
+
 _dirstack_ "$@"
diff --git a/zsh-edit.plugin.zsh b/zsh-edit.plugin.zsh
index e4aed2a..a2b1a70 100644
--- a/zsh-edit.plugin.zsh
+++ b/zsh-edit.plugin.zsh
@@ -5,9 +5,10 @@ setopt autopushd pushdminus
   typeset -gHa _edit_opts=( extendedglob NO_listbeep rcquotes warncreateglobal )
   setopt $_edit_opts
 
-  local fdir=${${(%):-%x}:h}/functions
+  local fdir=${${(%):-%x}:A:h}/functions
   unfunction bindkey 2>/dev/null
-  autoload -Uz $fdir/bindkey $fdir/_*~*.zwc
+  autoload -Uz $fdir/bindkey
+  autoload -Uz $fdir/_*~*.zwc(:t)
 
   local widget
   for widget in {forward,backward,kill,backward-kill}-word; do
@@ -20,11 +21,11 @@ setopt autopushd pushdminus
   bindkey -M emacs '^[Y' reverse-yank-pop
 
   zle -N _dirstack
-  bindkey -M emacs '^[`' _dirstack # Show dir stack.
+  bindkey '∂' _dirstack # Show dir stack.
 
   zle -N _pushd
-  bindkey -M emacs '^[-' _pushd # Go back one dir.
-  bindkey -M emacs '^[=' _pushd # Go forward one dir.
+  bindkey 'ß' _pushd # Go back one dir.
+  bindkey 'ƒ' _pushd # Go forward one dir.
 
   bindkey -M menuselect '^[-' menu-complete
   bindkey -M menuselect '^[=' reverse-menu-complete
